### ISSUE TITLE
The inline-comment scan runs a `//` comment on past a bare carriage return, where Less ends it

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#721]
Found by the review of the branch closing [#359](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/359), which made `declaration-block-trailing-semicolon` read the guard's answer as a reading of the file rather than only as a reason to decline a fix.

### What minimal example or steps are needed to reproduce the bug?

A semicolon behind a bare carriage return inside an end-of-line comment, where `<CR>` stands for the carriage return itself:

```less
a {
	color: pink // c<CR>;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-less",
  "rules": {
    "@stylistic/less/declaration-block-trailing-semicolon": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.less" --config myconfig.json --fix
```

### What did you expect to happen?

Nothing reported under `always`, and under `never` the semicolon reported and taken away: Less normalises `\r\n?` to `\n` before it parses (`parser.js` of `less@4.9.1`), so the carriage return ends the comment and the semicolon is code. Put `@@@` behind that semicolon on its line and Less refuses the file; put it behind a semicolon standing in the comment's text and Less compiles it.

### What actually happened?

`always` reports `Expected a trailing semicolon` and writes nothing, on every run; `never` reports nothing. The `semicolon-in-comment` sweep counts 8 such runs, 4 under each option, over the four nodes the reading of #359 covers. A semicolon behind a bare carriage return in the raws behind the node, `color: pink // ;<CR>	;`, is read as the comment's text the same way.

### What is the cause?

`endsWithInlineComment` closes a `//` comment on a line feed alone, and says so in its JSDoc; `InlineCommentReading` says which break closes a comment is PostCSS's question. The less namespace's `semicolonFlagIsCommentText` reads the flag of `postcss-less` through `writesIntoInlineComment`, and the rule reads the raws behind the node to the first line feed, so the carriage return that Less reads as a break is read as the comment's text there. Before #359 the rule believed the flag and was right on this spelling.

Whether the scan should read the breaks of the syntax it is handed, or the reading of #359 should ask differently, is the first question of the branch.

Measured against `less@4.9.1` and `postcss-less@6.0.0`.
